app/src/main/java/com/hedvig/app/feature/keygear/ui/itemdetail/viewbinders/NameBinder.kt

```diff
@@ -40,7 +41,18 @@ class NameBinder(
 
     fun bind(data: KeyGearItemQuery.KeyGearItem) {
         val name = data.fragments.keyGearItemFragment.name
-        root.name.text = name
+        val category =
+            root.context.resources.getString(data.fragments.keyGearItemFragment.category.label)
+
+        if (name.isNullOrEmpty() || name.isNullOrBlank()) {
+            root.nameEditText.setText(category)
```

Comment:
Is this desirable? I imagine that if you edit the 'non-filled' name, you'd like to start from a clean `EditText` 🙂

Reply:
that is true